src/exp_log_geo.jl

```diff
@@ -0,0 +1,109 @@
+@doc raw"""
+    exp(M::AbstractManifold, p, X)
+    exp(M::AbstractManifold, p, X, t::Real = 1)
+
+Compute the exponential map of tangent vector `X`, optionally scaled by `t`,  at point `p`
+from the manifold [`AbstractManifold`](@ref) `M`, i.e.
+
+```math
+\exp_pX = γ_{p,X}(1),
```

Comment:
```suggestion
\exp_p X = γ_{p,X}(1),
```
I think it will be more readable in REPL with a space there (I know this makes no difference to LaTeX rendering).

Reply:
Ah, sure, I did not think about REPL here.